pom.xml

```diff
@@ -56,6 +56,7 @@
         <dep.guice.version>7.0.0</dep.guice.version>
         <dep.errorprone.version>2.23.0</dep.errorprone.version>
         <dep.jackson.version>2.15.3</dep.jackson.version>
+        <dep.jackson-dataformat-csv.version>2.15.0</dep.jackson-dataformat-csv.version>
```

Comment:
The latest version for this is 2.15.3, which I'd assume matches all other Jackson components. Can we just use `dep.jackson.version` for this?

Reply:
Ups, absolutely. I went looking at git tags when I started on this and could only find a 2.15.0 for the csv part, hence I split that and figured they are doing separate bugfix lines.
But you are of course right, there is a 2.15.3. I'll change and rebase.